src/UI/admin/components/hall-request/AdminHallRequestStyle.js

```diff
@@ -11,4 +11,504 @@ const HallReqTitle = styled.h2`
   color: blue;
 `;
 
-export { HallReqContainer, HallReqTitle };
+const HallDiv = styled.div`
+  margin: auto;
+  width: 1024px;
+  height: 352px;
+  border-radius: 4px;
+  border: solid 1px #d9d9d9;
+  background-color: #ffffff;
+  display: flex;
+  box-sizing: border-box;
+`;
+
+const ImageDiv = styled.div`
+  height: 352px;
+  width: 268px;
+  display: flex;
+`;
+
+const TextDiv = styled.div`
+  height: 352px;
+  width: 504px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  box-sizing: border-box;
+`;
+
+const ButtonDiv = styled.div`
+  height: 352px;
+  width: 260px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  box-sizing: border-box;
+`;
+
+const ImageHall = styled.div`
+  width: 228px;
+  height: 136px;
+  border-radius: 2px;
+  border: solid 1px #229c7f;
+  margin-top: 16px;
+  margin-left: 16px;
+  object-fit: cover;
+`;
+const HeaderHall = styled.div`
+  width: 278px;
+  height: 32px;
+  font-family: Lora;
+  font-size: 20px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #000000;
+  margin-top: 18px;
+`;
+
+const AdressHall = styled.div`
+  width: 117px;
+  height: 20px;
+  opacity: 0.5;
+  font-family: Poppins;
+  font-size: 12px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #000000;
+  margin-top: 25px;
+  margin-left: 24px;
+`;
+
+const EventItem = styled.div`
+  width: 504px;
+  height: 20px;
+  border-radius: 2px;
+  background-color: #229c7f;
+  margin-top: 16px;
+  font-family: Poppins;
+  font-size: 12px;
+  font-weight: 600;
+  color: #ffffff;
+  margin-top: 16px;
+  padding: 2px 8px;
+  box-sizing: border-box;
+`;
+
+const EventName = styled.div`
```

Comment:
Zašto je EventName div? Kada mu daš takvo ime očekujem da je to nekakav tipografski element npr. h ili p. Molim te ispravi ovo